modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/bar-chart/index.css

```diff
@@ -0,0 +1,19 @@
+.bar-chart{
+  flex-wrap: wrap;
+}
+
+.bar-chart .bar-chart-title{
+  text-transform: uppercase;
+}
+
+.bar-chart .bb-chart-text{
+	color: #272833;
+	font-size: 30px;
+	font-weight: bold;
+}
```

Comment:
I believe that we have Dialect properties for this.

Reply:
The class in question is part of the Billboard.js library, so the CSS had to be written for this class, not being possible to use Dialect in this case.